qemu/tests/block_copy.py

```diff
@@ -143,9 +143,11 @@ def is_paused(self):
         """
         paused = self.get_status().get("paused")
         offset_p = self.get_status().get("offset")
-        time.sleep(random.uniform(1, 3))
-        offset_l = self.get_status().get("offset")
-        paused &= offset_p == offset_l
+        paused &= bool(utils_misc.wait_for(lambda: not self.get_status().get("busy"),
+                                           timeout=5))
+        if paused:
+            offset_l = self.get_status().get("offset")
+            paused &= offset_p == offset_l
```

Comment:
@yanan-fu right, but already wait for 5 seconds for busy=false, I don't prefer sleep between offset checking; if it can't pause within 5 seconds, I think it should be qemu bug. So I will modify but without time.sleep.